server/game/cards/09.1-FtE/AdornedBarcha.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes } = require('../../Constants');
+
+class AdornedBarcha extends DrawCard {
+    setupCardAbilities(ability) {
```

Comment:
Please use `AbilityDsl` for new cards instad of passing `ability` to the `setupCardAbilities` function.

Such as https://github.com/gryffon/ringteki/blob/975d2d967a4f9658b18b3e838f4fc31c832c3c68/server/game/cards/08-MotC/ArmamentArtisan.js#L2-L14